dbt_subprojects/tokens/models/prices/prices_trusted_tokens.sql

```diff
@@ -143,6 +143,10 @@ WITH trusted_tokens AS (
                 , ('solana', from_base58('mSoLzYCxHdYgdzU16g5QSh3i5K3z3KZK7ytfqcJm7So'))
                 , ('solana', from_base58('Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB'))
                 , ('solana', from_base58('EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v'))
+                , ('sonic', 0x039e2fB66102314Ce7b64Ce5Ce3E5183bc94aD38)
+                , ('sonic', 0x29219dd400f2Bf60E5a23d13Be72B486D4038894)
+                , ('sonic', 0x50c42dEAcD8Fc9773493ED674b675bE577f2634b)
+                , ('sonic', 0xe715cba7b5ccb33790cebff1436809d36cb17e57)
```

Comment:
```
select
    contract_address,
    count(1)
from
    erc20_sonic.evt_transfer
group by
    contract_address
order by
    2 desc
limit
    10
```
to establish initial trusted tokens, should compliment when we add DEXs to find gaps in amounts calculated